src/lib.rs

```diff
@@ -11,7 +11,17 @@ extern crate todo_txt;
 #[cfg(test)]
 #[macro_use]
 extern crate pretty_assertions;
+#[cfg(feature = "integration_tests")]
+#[macro_use]
+extern crate serde_derive;
 
 pub mod compute_changes;
 pub mod display_changes;
 pub mod stable_marriage;
+
+#[cfg(all(test, not(feature = "integration_tests")))]
+#[test]
+// #[ignore]
```

Comment:
It is there to make it easy to ignore this potentially annoying test without needing to go through the documentation to remember what the syntax is to ignore tests

Reply:
Oh, didn't think about it. In this case, maybe change the `#[cfg]` above to
```rust
#[cfg(all(test,
          not(feature="integration_tests"),
          not(feature="ignore_integration_tests_warning")))]
```
and adapt the message in the `panic` to mention it? This would make it even easier to ignore this check :)